runtime/nvqir/custatevec/CuStateVecCircuitSimulator.cu

```diff
@@ -552,10 +551,8 @@ public:
     // Use custatevec accessor to retrieve the view
     custatevecAccessorDescriptor_t accessor;
     const uint32_t nIndexBits = std::log2(stateDimension);
-    // Note: we use MSB bit ordering when reporting the state vector
-    // hence, bit ordering vector = [N-1, N-2, ..., 0]
     std::vector<int32_t> bitOrdering(nIndexBits);
-    std::iota(std::rbegin(bitOrdering), std::rend(bitOrdering), 0);
+    std::iota(std::begin(bitOrdering), std::end(bitOrdering), 0);
```

Comment:
I think Thien mentioned that this implementation used the accessor functions from custatevec because he had issues getting the bit ordering right. Do you think you could try replacing these calls with a simple call to `cudaMemcpy` with type `cudaMemcpyDeviceToHost` and see if you get the correct ordering your are looking for? I think this memcpy would ultimately be optimal.